web/app/views/index.scala.html

```diff
@@ -699,10 +700,20 @@
             {{#dataset-access hasAccess=hasAccess accessibilities=accessibilities}}
             {{/dataset-access}}
           </div>
-          <div id="compliancetab" class="tab-pane">
-            {{#dataset-compliance hasCompliance=hasCompliance compliance=compliance}}
-            {{/dataset-compliance}}
-          </div>
+            <div id="compliancetab" class="tab-pane">
+                {{#dataset-compliance securitySpec=securitySpec
```

Comment:
It seems you don't need to use the block form for this component. Same for other components throughout this pull request.